lib/relationships.js

```diff
@@ -130,3 +161,17 @@ function updateRelation (model, id, data) {
     }
   })
 }
+
+function keyByModel(assocModel, model) {
+  var key = null;
+  _.each(assocModel.relations, function(relation){
+    if (relation.modelTo.modelName === model.modelName) {
+      key = relation.keyFrom;
+    }
+  })
+  
+  if (key === null) {
+    console.log('Can not find relation for ' + model.modelName);
```

Comment:
When would key === null?
Should this log? Or throw error? Or silently do nothing?